natlib/svcd.c

```diff
@@ -93,8 +95,13 @@ static int svcd_init( lua_State *L )
     lua_getglobal(L, "SVCD");
     printf("Put table at %d\n", lua_gettop(L));
 #endif
-    //Now begins the part that corresponds with the lua init function
+    
+    //Override symbols in the lua table with C implementation
+    lua_pushstring(L, "ndispatch");
+    lua_pushlightfunction(L, svcd_ndispatch);
+    lua_settable(L, 3); 
 
```

Comment:
It doesn't seem necessary to overwrite with the C implementation here.
Holy Cow